cmd/root.go

```diff
@@ -57,23 +57,34 @@ func init() {
 }
 
 func initConfig() {
+	viper.AutomaticEnv()
```

Comment:
is there a reason this was moved above config file loading? I'm not familiar with what this does, but if possible, I'd like to keep this PR scoped to just the config file changes

Reply:
The call to `viper.AutomaticEnv()` was originally done before the call to `viper.ReadInConfig()` and since I moved the latter to within the if/else logic, I moved the  `viper.AutomaticEnv()` higher up to avoid duplicating it.  But I can keep it closer to `viper.ReadInConfig()` by coding it twice.